### ISSUE TITLE
Mouse driver problem

### ISSUE THREAD

**Issue by darekz08** [AlgorithMan-de/wyoos#20]
In virtual box driver works good, in qemu cursor is "jumping", but in vmware and on real hardware it does't do anything. How to fix that?

mouse.h: https://pastebin.com/AjbxGS7R
mouse.cpp: https://pastebin.com/jqC959dc

**Comment by yeppiidev** [comment]
Same problem

**Comment by Kap0001** [comment]
just shut your ass up and dont make an OS

just kidding lol i have the same problem, anyone know how to fix this?

**Comment by yeppiidev** [comment]
i wrote another mouse driver in C, which actually works. unfortunately the source files for the project which had that driver got lost while I switched computers. Let me see if I can improve the current driver in this repository!

**Comment by PureBulb** [comment]
my problem is the os in vmware cursor is "jumping"

**Comment by bberrevoets** [comment]
@yeppiidev 
Did you find some solution for this, or are you not busy with this anymore.

**Comment by yeppiidev** [comment]
yup, left it a long time ago. afaik i do remember getting mouse work but not on this kernel (it was another os project written in C)

**Comment by bberrevoets** [comment]
Thanks for the reply.
In the meantime, I got it working 😊

**Comment by yeppiidev** [comment]
would you mind sharing the code?

**Comment by bberrevoets** [comment]
Hi,

When i look at the code that is on github, (https://github.com/AlgorithMan-de/wyoos/commit/48959b002f33a1e1428cd190da8c268929f2aee5),
The only difference is that I have set the offset to 2 in the constructor instead of 0.
